packages/shared/src/modules/activities/libs/types/activity-finish-section-request-dto.type.ts

```diff
@@ -0,0 +1,11 @@
+type ActivityFinishSectionRequestDto = {
+	course: {
+		id: null | number;
```

Comment:
```suggestion
		id: null | number;
```
it really can be null?